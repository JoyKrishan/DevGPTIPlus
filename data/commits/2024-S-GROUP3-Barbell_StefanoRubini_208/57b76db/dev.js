    presets: [['babel-preset-expo', { jsxImportSource: 'nativewind' }], 'nativewind/babel'],
      /* 50 */
      'react-native-reanimated/plugin',
      /* 49 */
      // ['module:react-native-dotenv'],
      // require.resolve('expo-router/babel'),
      // require.resolve('react-native-reanimated/plugin'),