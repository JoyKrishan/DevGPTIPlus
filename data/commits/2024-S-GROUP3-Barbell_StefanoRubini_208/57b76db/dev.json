    "@ianvs/prettier-plugin-sort-imports": "^4.1.1",