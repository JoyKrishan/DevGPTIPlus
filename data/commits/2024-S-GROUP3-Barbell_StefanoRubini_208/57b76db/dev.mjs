  // prettier
  semi: false,
  printWidth: 100,
  singleQuote: true,
  quoteProps: 'as-needed',
  jsxSingleQuote: true,

  plugins: ['@ianvs/prettier-plugin-sort-imports', 'prettier-plugin-tailwindcss'],
  tailwindConfig: fileURLToPath(new URL('../../tooling/tailwind/web.ts', import.meta.url)),
  // https://github.com/tailwindlabs/prettier-plugin-tailwindcss