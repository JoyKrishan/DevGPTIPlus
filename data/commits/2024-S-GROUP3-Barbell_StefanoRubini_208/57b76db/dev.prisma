  workoutHistory       WorkoutLog[]
  customExercises      Exercise[]
// * friendId: The identifier for the friend, used to query friend info.

  friendId Int
model WorkoutLog {
  id            Int          @id @unique @default(autoincrement())
  name          String       @db.Char(100)
  createdAt     DateTime     @default(now())
  updatedAt     DateTime     @updatedAt
  likes         Int          @default(0)
  past_workouts WorkoutLog[]
  id        Int       @id @unique @default(autoincrement())
  name      String    @db.Char(100)
  workout   Workout[]
  User      User?     @relation(fields: [userId], references: [id])
  userId    Int?
  createdByUserId Int