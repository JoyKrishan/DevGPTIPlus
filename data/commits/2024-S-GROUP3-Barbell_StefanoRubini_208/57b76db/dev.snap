    className="text-center  text-white text-base"